worker/firewaller/firewaller.go

```diff
@@ -943,7 +942,7 @@ func (fw *Firewaller) ingressRulesForExposedMachineUnit(machine *machineData, un
 }
 
 // TODO(wallyworld) - consider making this configurable.
-const maxAllowedCIDRS = 20
+const maxAllowedCIDRS = 30
```

Comment:
This was increased because we now have one less option for fallback before going completely public.

Reply:
Even so, I'm not sure anyone used that fall-back, so it's probably not necessary.